test/ZendeskApi_v2.Test/TicketTests.cs

```diff
@@ -1119,6 +1119,162 @@ public void CanImportTicketAsync()
+            var sourceMergeComment = $"Closing in favor of {targetTicketId}";
+
+            var res = api.Tickets.MergeTickets(
+                targetTicketId, 
+                mergeIds, 
+                targetMergeComment, 
+                sourceMergeComment);
+
+            Assert.That(res.JobStatus.Status, Is.EqualTo("queued"));
+
+            do
+            {
+                Thread.Sleep(5000);
+                var job = api.JobStatuses.GetJobStatus(res.JobStatus.Id);
+                Assert.That(job.JobStatus.Id, Is.EqualTo(res.JobStatus.Id));
+
+                if (job.JobStatus.Status == "completed") break;
+            } while (true);
+
+            var counter = 0;
+            foreach (var id in mergeIds)
+            {
+                var oldTicket = api.Tickets.GetTicket(id);
+                Assert.That(oldTicket.Ticket.Id.Value, Is.EqualTo(id));
+                Assert.That(oldTicket.Ticket.Status, Is.EqualTo("closed"));
+
+                var oldComments = api.Tickets.GetTicketComments(id);
+                Assert.That(oldComments.Comments.Count, Is.EqualTo(2));
+                Assert.That(oldComments.Comments[0].Body, Is.EqualTo(sourceDescription[counter]));
+                Assert.That(oldComments.Comments[1].Body, Is.EqualTo(sourceMergeComment));
+
+                api.Tickets.DeleteAsync(id);
+                counter++;
+            }
+
+            var ticket = api.Tickets.GetTicket(targetTicketId);
+            Assert.That(ticket.Ticket.Id.Value, Is.EqualTo(targetTicketId));
+
+            var comments = api.Tickets.GetTicketComments(targetTicketId);
+            Assert.That(comments.Comments.Count, Is.EqualTo(2));
+            Assert.That(comments.Comments[0].Body, Is.EqualTo(targetDescription));
+            Assert.That(comments.Comments[1].Body, Is.EqualTo(targetMergeComment));
+
+            api.Tickets.DeleteAsync(targetTicketId);
+        }
+
+        [Test]
+        public void CanMergeTicketsAsync()
+        {
+            var sourceDescription = new List<string> { "This is a source ticket #1", "This is a source ticket #2" };
+            var targetDescription = "This is a the target ticket";
+
+            var sourceTicket1 = new Ticket
+            {
+                Subject = "Source Ticket 1",
+                Comment = new Comment { Body = sourceDescription[0], Public = true, }
+            };
+            var sourceTicket2 = new Ticket
+            {
+                Subject = "Source Ticket 2",
+                Comment = new Comment { Body = sourceDescription[1], Public = true, }
+            };
+            var targetTicket = new Ticket
+            {
+                Subject = "Target Ticket",
+                Comment = new Comment { Body = targetDescription, Public = true, }
+            };
+
+            var mergeIds = new List<long>();
+
+            var tick = api.Tickets.CreateTicket(sourceTicket1);
```

Comment:
```suggestion
            var tick = await api.Tickets.CreateTicketAsync(sourceTicket1);
```

Reply:
Did not seem any reason to call everything else Async just to test the Merge Async call.  Again, followed the existing code as a guideline.

Reply:
Can you please share where you see that an async is only using async for part of the test?